### ISSUE TITLE
Timeline doesn't update

### ISSUE THREAD

**Issue by helioshk** [Helioviewer-Project/JHelioviewer-SWHV#186]
Hi
After adding a timeline (for example XRS-B), the timeline seems to never get updated anymore. For example if i leave jhelioviewer running for 2 hours, no newer data is plotted, even when removing the timeline and adding it again.

**Comment by bogdanni** [comment]
That's a feature normally :)
There's response caching at the HTTP client level. The server probably doesn't use or modify the Last-Modified tag (and it's impossible/difficult for it to compute such a timestamp).
Solution to be investigated.

**Comment by bogdanni** [comment]
There is a new build at http://swhv.oma.be/download_test/
Does it solve the issue?

**Comment by helioshk** [comment]
@bogdanni Still the same, the timeline doesn't update, as far as I can tell. 

When should it update? For me it would be ok, if the timeline would be updated, every time the image layer gets changed/synched/updated.

**Comment by bogdanni** [comment]
I dug deeper and I'm afraid that the problem is more complex than a simple network cache.
I will have to rework the design of the code. In the meanwhile, the only solution is to close and restart JHV.

**Comment by bogdanni** [comment]
I agree that the behavior you are seeking is the desired one, unfortunately the code was not designed for that.

**Comment by helioshk** [comment]
Ok, thanks for having a look at it. It's not that important for me. I can work around it.
